src/common/types/navigation/params/app-stack-param.ts

```diff
@@ -9,6 +9,7 @@ type AppStackParamList = {
     isTeamOkr: boolean;
   };
   [AppRoute.ADD_KEY_RESULT]: undefined;
+  [AppRoute.ONBOARDING_SETUP]: undefined;
```

Comment:
better use `void`

Reply:
```typescript
type void = null | undefined
```

Reply:
Common practice here is to use `undefined`. We can leave it as is